WcaOnRails/app/views/incidents/_incident_competition_fields.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="nested-fields">
+  <%= f.input :competition_id, as: :incident_competition_id, only_one: true, input_html: { class: "competition-selector" }, link_html: link_to_remove_association(button_tag(icon("times"), type: "button", class: "btn btn-danger"), f) %>
```

Comment:
I couldn't manage to align the three elements correctly (comp id input, comment input, and the "times" icon).
I can't remember exactly the details but I struggled to have them on one line and nicely aligned (maybe because they were not in the same form group), so I gave up and just put them altogether in a new input.